linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/ResponseClassifierConfig.scala

```diff
@@ -0,0 +1,11 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.buoyant.h2.service.ResponseClassifier
+import io.buoyant.config.{ConfigInitializer, PolymorphicConfig}
+
+abstract class ResponseClassifierInitializer extends ConfigInitializer
+abstract class ResponseClassifierConfig extends PolymorphicConfig {
```

Comment:
For most dynamically loaded plugins we list all the plugin types in Linkerd.scala in linkerd-core.  However, we can't do that for this one since it's H2 specific and we don't want linked-core to depend on linkerd-protocol-h2.  The easiest thing to do for now is to explicitly list all subclasses here with a @JsonSubTypes annotation.  This is what we do for LoadBalancerConfig, for example.